src/v/config/node_config.cc

```diff
@@ -142,6 +142,13 @@ node_config::node_config() noexcept
       "operator intervention is needed to startup the broker.",
       {.visibility = visibility::user},
       std::nullopt)
+  , large_allocation_warning_threshold(
+      *this,
+      "large_allocation_warning_threshold",
```

Comment:
It's good to make the first word of a propertya noun that could group things logically when sorting alphabetically, even if right now there's just one memory-related property here.

Maybe this could just be `memory_allocation_warning_threshold`?

Reply:
Good point, will switch it to the recommended name.